package.json

```diff
@@ -46,13 +43,14 @@
     "h264-profile-level-id": "^1.0.0",
     "netstring": "^0.3.0",
     "random-number": "^0.0.9",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "gulp": "^4.0.2",
+    "gulp-clang-format": "^1.0.27",    
```

Comment:
"gulp-clang-format" must be in "devDependencies" (as it was).

Reply:
yep.  "gulp-clang-format" already moved into "devDependencies".